tests/js/server/utils-tests.js

```diff
@@ -0,0 +1,154 @@
+/*
+
+  Tests for the gpii.express.user.utils component.
+
+ */
+/* eslint-env node */
+"use strict";
+
+var fluid        = require("infusion");
+var gpii         = fluid.registerNamespace("gpii");
+
+require("../../../");
+require("../lib/");
+
+var jqUnit       = require("node-jqunit");
+
+fluid.registerNamespace("gpii.tests.express.user.utils.caseHolder");
+
+gpii.tests.express.user.utils.createUser = function (utils, sequenceDelay) {
+    var prom = utils.createNewUser({
+        username: "myFirstUser",
+        password: "this is a password",
+        email: "myFirstUser@gpii.net"
+    });
+    prom.then(function (data) {
+        jqUnit.assertEquals("Generated Couch ID", "org.couch.db.user:myFirstUser", data._id);
+        jqUnit.assertEquals("Email", "myFirstUser@gpii.net", data.email);
+        jqUnit.assertEquals("Username", "myFirstUser", data.username);
+        jqUnit.assertTrue("Unlock the password", gpii.express.user.utils.verifyPassword(data, "this is a password"));
+        sequenceDelay.events.onComplete.fire();
```

Comment:
There's no need for any of this stuff, the IoC Testing framework natively supports promise-returning fixtures: https://docs.fluidproject.org/infusion/development/IoCTestingFramework.html#supported-fixture-records

Reply:
This is updated with Tasks now and pushed up.  Cheers for the link

Reply:
It is?  I don't see any use of tasks in the latest version of these tests.

Reply:
OK, I see what you did.  I guess I was expecting you to eliminate the wrapper function in favour of better reject and resolve handler definitions.